docs/guides/bulk/download.md

```diff
@@ -0,0 +1,32 @@
+---
+heading: Bulk
+seo: Bulk | Bulk Download | How to Use Bulk Download
+title: Download
+description: How to use Bulk Download
+layout: sidebarleft
+apis: API Docs
+breadcrumbs: /docs/guides/home.html
+parent: Back to Guides
+order: 2
+---
+
+# Bulk Download
+
+The Bulk download can be used to generate a csv or json file from a large number of records. Since the bulk download works asynchronously, two calls are required to download a file.
+
+### 1. Create a Bulk Job
+The `POST /bulk/query` creates a bulk job, which will build out a file in Cloud Elements. The records included in the file are based on the CEQL query used to generate the job. Since the job will take an unknown amount of time to complete, a callback url can also be included in the bulk job. When the job completes it will make a callback to the url. The query is composed of several parts:
+
+- Headers:
+  - `Elements-Async-Callback-Url`: The webhook to be notified when a job has completed
+- Query:
+  - `q` : A CEQL query for records to be included in the bulk job, i.e. `select * from contacts`
+
+### 2. Download the Bulk File
+Once the job generated from the `/bulk/query` call has completed, the `GET /bulk/{id}/{objectName}` can be used to download the file. The file can be downloaded as a `csv` or `json` file.
```

Comment:
~~Once~~ After the job generated from the `/bulk/query` call ~~has completed~~completes, ~~the~~use `GET /bulk/{id}/{objectName}` ~~can be used~~ to download the file~~. The file can be downloaded~~ as a ~~`csv` or `json`~~ csv or json file.

Once isn't wrong, but I follow the rule that you should use the word with the fewest meanings to avoid confusion. Again, this is more for non-native speakers.
I removed the code formatting for csv or json because of 2 reasons. 1 was in the first paragraph, they got no special formatting and 2 is that we should define how to refer to formats and it should be different from code, which should also be different from UI widgets. I'm going to add to the next sprint a quick style guide development so something like this can go in.